README.md

```diff
@@ -480,6 +480,10 @@ The content still gets escaped properly, with the exception of the `script` and
 
 ## Changelog
 
+2.0.0:
```

Comment:
@boutell Technically this PR breaks backwards compatibility, so we should bump the major version in case anyone is relying on the behaviour I've changed.

Reply:
Re: _I'm not sure, however, where the bc break is if it's optional._

If someone was already using `parser.decodeEntities = false`, and what they get back from `sanitiseHtml()` changed from what they were expecting, I would say that breaks backwards compatibility.

E.g. this which came to light about an hour after my very first open source PR was merged despite me having mention possible BC breaking...

https://github.com/Behat/Transliterator/pull/18#commitcomment-21637919